src/main/java/com/shapesecurity/salvation/data/Policy.java

```diff
@@ -420,7 +502,18 @@ public String show() {
 		return sb.toString();
 	}
 
-	private boolean defaultsAllowAttributeWithHash(@Nonnull HashAlgorithm algorithm, @Nonnull Base64Value hashValue) {
+	private boolean defaultsAllowScriptAttributeWithHash(@Nonnull HashAlgorithm algorithm, @Nonnull Base64Value hashValue) {
```

Comment:
We could just deduplicate this and `defaultsAllowStyleAttributeWithHash` and call it `defaultsAllowAttributeWithHash`.